tests/resources/functions/java/Index.java

```diff
@@ -0,0 +1,55 @@
+import java.util.Map;
```

Comment:
We discussed to not include java & swift since they are extremely slow to build.
Let's remove the Java test